packages/cdktf-cli/test/config.test.ts

```diff
@@ -211,19 +214,544 @@ describe("parseConfig", () => {
           "terraformModules": Array [
             TerraformModuleConstraint {
               "fqn": "terraform-aws-modules/vpc/aws",
-              "name": "aws",
+              "name": "vpc",
+              "namespace": "terraform-aws-modules/aws",
               "source": "terraform-aws-modules/vpc/aws",
               "version": "2.39.0",
             },
             TerraformModuleConstraint {
               "fqn": "customAWSVpc",
               "name": "customAWSVpc",
+              "namespace": undefined,
               "source": "https://github.com/terraform-aws-modules/terraform-aws-vpc",
               "version": "~> v2.0",
             },
           ],
         }
       `);
     });
+
+    it("parses local path", async () => {
```

Comment:
All these tests are really cool, seems like the implementation needs to be rock solid to make these pass